manifest.json

```diff
@@ -3787,6 +3834,75 @@
+            "website": "https://github.com/Banane9/NeosCorrectGizmoScaling",
+            "sourceLocation": "https://github.com/Banane9/NeosCorrectGizmoScaling",
+            "authors": {
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/Banane9/NeosCorrectGizmoScaling/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosCorrectGizmoScaling/releases/download/v1.0.0/CorrectGizmoScaling.dll",
+                            "sha256": "749B222F8FE36B10985E2AA1076E649CA72F0EF1B2D311D00FC7DF3D83C8D031"
+                        }
+                    ]
+                }
+            }
+        },
+        "Banane9.BoundedUIX": {
+            "name": "BoundedUIX",
+            "description": "Makes UIX elements selectable and editable with the usual Slot gizmos, as well as fixing UIX elements breaking when a parent is inserted.",
+            "category": "Developers",
+            "website": "https://github.com/Banane9/NeosBoundedUIX",
+            "sourceLocation": "https://github.com/Banane9/NeosBoundedUIX",
+            "authors": {
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "2.2.2": {
+                    "releaseUrl": "https://github.com/Banane9/NeosBoundedUIX/releases/tag/v2.2.2",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosBoundedUIX/releases/download/v2.2.2/BoundedUIX.dll",
+                            "sha256": "7B2FEEBC1A6EB82AC65455AFB060C2DA7328B3E651805E4A2EEFB3DA210140F1"
+                        }
+                    ]
+                }
+            }
+        },
+        "Banane9.ComponentAttacherSearch": {
+            "name": "ComponentAttacherSearch",
+            "description": "Adds a search bar to component attachers.",
+            "category": "Developers",
+            "website": "https://github.com/Banane9/NeosComponentAttacherSearch",
+            "sourceLocation": "https://github.com/Banane9/NeosComponentAttacherSearch",
+            "authors": {
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/Banane9/NeosComponentAttacherSearch/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosComponentAttacherSearch/releases/download/v1.0.0/ComponentAttacherSearch.dll",
+                            "sha256": "FFB3DC6AAAB2E18795A9B8200F01AE3B370087D22D754D47F24B3DAA515E35E5"
```

Comment:
Wrong Hash
Expected:
ffb3dc6aaab2e18795a9b8200f01ae3b370087d22d754d47f24b3daa515e35e5
Got:
6a0a7e225307add0fb4a704edf92290fef1a406f5ecb8a111e05fa97993e3a8d

Reply:
hmmm, I forgot to update the hash from the one I copied... but also it wasn't the hash of the release build - weird.
I updated the hash and file, so it should match now